app/channels/iteration_channel.rb

```diff
@@ -0,0 +1,16 @@
+class IterationChannel < ApplicationCable::Channel
+  def subscribed
+    stream_from CHANNEL_NAME
```

Comment:
Why change from `stream_from` to `stream_for`?

Reply:
I took that from https://guides.rubyonrails.org/action_cable_overview.html#client-server-interactions. The way I read the docs, was that `stream_for` related to objects. But I might have misread

Reply:
I've checked this locally, and `stream_for` does indeed _not_ work. `stream_from` does.